src/test/java/org/openrewrite/java/testing/jmockit/JMockitExpectationsToMockitoTest.java

```diff
@@ -23,8 +23,7 @@
 import org.openrewrite.test.RecipeSpec;
 import org.openrewrite.test.RewriteTest;
 
-class JMockitToMockitoTest implements RewriteTest {
-
+class JMockitExpectationsToMockitoTest implements RewriteTest {
```

Comment:
@timtebeek I named it `JMockitToMockitoTest` because we're actually running the `org.openrewrite.java.testing.jmockit.JMockitToMockito` recipe.